packages/selleo-design-core/src/parser.tsx

```diff
@@ -16,10 +16,11 @@ const buildComponent = (html: string) => (): VNode => {
 export const parseComponent = (html: string) => {
   const Component = buildComponent(html);
   const HTMLString = render(<Component />, null, { pretty: true });
+  const HTMLStringResult = HTMLString.replaceAll(/><\/input>|><\/img>/g, '>');
```

Comment:
nice, GJ with it! :D 
Can we also replace in ReactString to be self-closing?

Reply:
Thanks! 😁
Yes, already changed :)